iron-selectable.html

```diff
@@ -264,7 +267,8 @@
     },
 
     _valueForItem: function(item) {
-      return item[this.attrForSelected] || item.getAttribute(this.attrForSelected);
+      return item[Polymer.CaseMap.dashToCamelCase(this.attrForSelected)] ||
```

Comment:
Yeah, this is going to repeat a lot of work otherwise; fix coming in just a moment.